packages/react/src/next-ssr-plugin.tsx

```diff
@@ -18,7 +19,7 @@ export function NextSSRPlugin(props: { routerConfig: EndpointMetadata }) {
   useServerInsertedHTML(() => {
     const html = [
       // Hydrate routerConfig on client globalThis
-      `globalThis.__UPLOADTHING = ${JSON.stringify(props.routerConfig)};`,
+      `globalThis.__UPLOADTHING = ${JSON.stringify(props.routerConfig, safeNumberReplacer)};`,
```

Comment:
may cause mismatch?

Reply:
Why if the fetch call also uses the replacer?